src/init.cpp

```diff
@@ -499,6 +499,9 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-torcontrol=<ip>:<port>", strprintf(_("Tor control port to use if onion listening enabled (default: %s)"), DEFAULT_TOR_CONTROL));
     strUsage += HelpMessageOpt("-torpassword=<pass>", _("Tor control port password (default: empty)"));
     strUsage += HelpMessageOpt("-upnp", strprintf(_("Use UPnP to map the listening port (default: %u)"), DEFAULT_UPNP));
+#ifdef USE_NATPMP
+    strUsage += HelpMessageOpt("-natpmp", strprintf("Use NAT-PMP to map the listening port (default: %s)", DEFAULT_NATPMP ? "1 when listening and no -proxy" : "0"));
```

Comment:
I've actually got some upstream work in the pipeline that removes translation from all these help messages for a couple reasons:

1) they are only ever translated for the qt client's help output.
2) most of them are highly technical in nature, and translation errors could lead to confusion or even improper usages. 

and (for our own reasoning)

3) by not translating these messages, we GREATLY reduce the number of strings that need translating, which will make it easier for translators to reach the 80% translated threshold that we require in order to even include translated strings for any particular language

so I decided to leave it as a non-translatable string here, rather than marking it for translation just to be un-marked in a later PR